kmk/modules/layers.py

```diff
@@ -143,3 +165,41 @@ def _print_debug(self, keyboard):
         # debug(f'__getitem__ {key}')
         if debug.enabled:
             debug(f'active_layers={keyboard.active_layers}')
+
+    def get_combo_layer(self, keyboard):
+
+        acvlyrs = []
```

Comment:
Please use full english words for variable names. The 4 bytes of saved space aren't worth the inconvenience of decyphering it.

Reply:
Fixed this as well.